### ISSUE TITLE
Could not install phalcon3

### ISSUE THREAD

**Issue by HubertSadecki** [shivammathur/setup-php#413]
**Describe the bug**
When trying to install phalcon3 within php 7.3, installation fail and verbose output is as follow:
```
E: Unable to locate package php7.3-phalcon3
E: Couldn't find any package by glob 'php7.3-phalcon3'
E: Couldn't find any package by regex 'php7.3-phalcon3'
[apt-fast 10:28:53]Package manager quit with exit code.
✗ phalcon Could not install phalcon on PHP 7.3.27
Error: The process '/usr/bin/bash' failed with exit code 1
```

The same thing happens when trying to install it with php 7.4. Probably it can happen with other PHP version I didn't verify.

Last time it worked fine was on 4th of February 2021 14:35 CET.

**Version**
- [ ] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [x] `v1`
- [x] `2.9.0`
- [x] `2.8.0`
- [x] `2.7.0`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
`ubuntu-latest`

**PHP versions**
- [x] `7.3`
- [x] `7.4`

**To Reproduce**
```
name: Commit review
on: [pull_request]
jobs:
  check:
    name: Review
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2

      - name: Setup PHP with composer and extensions
        uses: shivammathur/setup-php@v2
        with:
          php-version: '7.3'
          extensions: mbstring, dom, zip, phalcon3, pgsql
        env:
          fail-fast: true
```

**Comment by shivammathur** [comment]
@HubertSadecki `ppa:ondrej/php` has dropped support for phalcon3. Will be fixed in next release tomorrow.

**Comment by shivammathur** [comment]
Fixed in `2.10.0` release.
